x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -29,3 +29,28 @@ description = "test built-in modulo math functions"
 # This query give a different result with ES EQL implementation because it doesn't convert to float data types for division
 expected_event_ids = [82, 83]
 query = "file where serial_event_id / 2 == 41"
+
+# Additional EQL queries with arithmetic operations that were not part of the original EQL implementation
```

Comment:
It would be worth adding these new tests to the original EQL implementation, as well.